src/main/java/org/folio/linked/data/service/resource/ResourceMarcServiceImpl.java

```diff
@@ -273,7 +273,7 @@ private Long save(org.folio.ld.dictionary.model.Resource modelResource) {
       throw new AlreadyExistsException(format("MARC record having srsID %s is already imported to data graph", srsId));
     }
 
-    return createResource(resourceModelMapper.toEntity(modelResource));
+    return saveAndPublishEvent(resourceModelMapper.toEntity(modelResource), ResourceUpdatedEvent::new);
```

Comment:
while working on karate tests I stumbled upon such check
![image](https://github.com/user-attachments/assets/a50cf385-37fa-4a53-95db-b326850cd38f)

therefore I've changed ResourceCreatedEvent to ResourceUpdatedEvent

Reply:
We have to publish `ResourceCreatedEvent` @askhat-abishev. 
Publishing `ResourceUpdatedEvent` here will be incorrect. Let me try to check the problem today.

Reply:
Got it Askhat. Let us update the karate test to ensure that the source of the record is switched from MARC to linked-data after the import.